lab-edward-smith/lib/readFilesCli.js

```diff
@@ -0,0 +1,20 @@
+const readFiles = require('../lib/readFiles');
+
+
+const files = process.argv.slice(2);
+const fileArray = [];
+
+var promise = readFiles.readFile(files[0]);
```

Comment:
I like your use of promises to try to handle the async, the problem is that the implementation isn't quite right. While you are guaranteeing that your content will come back in the right order, you don't start reading the next file until the contents from the previous one have returned, which essentially means that your process is synchronous. You want to kick off all of your file reads at the same time.